projects/setuptools.cmake

```diff
@@ -0,0 +1,15 @@
+#add_custom_target(setuptools)
+#
+#add_custom_command(COMMAND ${PYTHON_EXECUTABLE_PREFIXED} -m pip install -U setuptools
+#                   TARGET setuptools)
+
+ExternalProject_Add(setuptools
```

Comment:
This should not be necessary - Python's build process includes an "ensure_pip" step that ensures pip and setuptools are included. However, it is not very verbose about it when it fails.